### PR TITLE
feat: add product pages

### PR DESCRIPTION
**Pull request by daywiss**
Signed-off-by: David <david@umaproject.org>

This adds 6 product pages for desktop and mobile views. To QA go to the product link in nav bar and click any of the specific products. 

This is missing the animations showing the Hero text, we will need to do this as a separate pr as it seems pretty tricky.

### CODE REVIEW

File: pages/products/success-tokens.tsx [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+      },
+      {
+        image: {
+          src: "/images/products/UMA-Shapeshift.svg",
+          alt: "Shapeshift",
+        },
+        title: "Shapeshift Proposal",
+        link: {
+          text: "Read it here",
+          href: "https://docs.google.com/spreadsheets/d/1wRYgPVIjEyk6ADi1f0IKhKXoN3iaZRG7v3Iza-HhNIw/edit#gid=53260911",
```
Comment (Tommy1231232) [inline pages/products/success-tokens.tsx:137]: Update this link to: https://forum.shapeshift.com/t/success-token-proposal-discussion/599

**Comment by amateima** [comment]
I think https://github.com/tameemsafi/typewriterjs is the easiest way to add the typewriting effect

**Review by Gamaranto — APPROVED** [review]

File: public/images/products/UMA-Sherlock.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+</svg>
```
[the remark this answers is not in the document]
Comment (daywiss) [reply public/images/products/UMA-Sherlock.svg:4]: i moved logo images there and re-used them if they already exists

File: components/ProductPageTemplate/components.tsx [hunk 3]
```diff
@@ -0,0 +1,547 @@
...
+  }
+`;
+
+export const GettingStartedCard = styled.div`
+  text-align: center;
+  background-color: var(--white);
+  border-radius: 5px;
+  padding: 16px;
+  display: flex;
+  margin: 8px 0;
```
[the remark this answers is not in the document]
Comment (daywiss) [reply components/ProductPageTemplate/components.tsx:388]: good to know

File: components/ProductPageTemplate/components.tsx [hunk 4]
```diff
@@ -0,0 +1,547 @@
...
+  @media ${QUERIES.tabletAndUp} {
+    margin: 16px auto;
+    padding: 0;
+    max-width: 800px;
+  }
+`;
+export const BasicContentLink = styled(PrimaryLink)`
+  width: auto;
+  font-size: 1.1rem;
+  padding 16px 0;
```
[the remark this answers is not in the document]
Comment (daywiss) [reply components/ProductPageTemplate/components.tsx:168]: fixed

File: pages/products/kpi-options.tsx [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+    ],
+  },
+  {
+    type: Section.Overview as Section.Overview,
+    title: "What is it?",
+    image: {
+      src: "/images/products/kpi-mobile-img2.png",
+      alt: "Example KPI UI",
+      label:
+        "Example UI for claiming and viewing KPI Options tokens for your community",
```
Comment (Tommy1231232) [inline pages/products/kpi-options.tsx:69]: Screen Shot 2022-03-18 at 1 59 46 PM


Lets update the color of the font "Example UI...." to that of the above designs

File: pages/products/kpi-options.tsx [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+      href: "https://docs.umaproject.org/kpi-options/summary",
+    },
+  },
+  {
+    type: Section.Starting as Section.Starting,
+    title: "How does it work?",
+    sections: [
+      {
+        title: "Define Target",
+        details: "Define the KPI (Key Perfomance Index) you want to hit.",
```
Comment (Tommy1231232) [inline pages/products/kpi-options.tsx:86]: Update to "Key Performance Index (KPI)"

File: pages/products/kpi-options.tsx [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+      },
+      {
+        title: "Create the KPI Options",
+        details: "Lock your project token and mint the KPI Option tokens.",
+      },
+      {
+        title: "Distribute and Market",
+        details:
+          "The tokens to your target community and advertise your campaign.",
```
Comment (Tommy1231232) [inline pages/products/kpi-options.tsx:95]: Lowercase the "T" in "The"

File: pages/products/success-tokens.tsx [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+    image: {
+      src: "/images/products/powered_by_uma.png",
+      alt: "UMA_logo",
+    },
+    carousel: {
+      prefix: "Success tokens empowers your DAO to",
+      variations: [
+        "raise funds for operations",
+        "attract strategic investors",
+        "put vesting agreements on chain",
```
Comment (Tommy1231232) [inline pages/products/success-tokens.tsx:18]: update to on-chain

File: pages/products/success-tokens.tsx [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+        title: "Strategic partners",
+        details: "Leverage the networks and resources of startup veterans.",
+        image: {
+          src: "/images/products/UMA-strategic-partners-65x65.svg",
+          alt: "Strategic partners",
+        },
+      },
+      {
+        title: "DeFi Native Raise",
+        details: "No lawyers or off-chain brokers required.",
```
Comment (Tommy1231232) [inline pages/products/success-tokens.tsx:36]: Remove the period

File: pages/products/range-tokens.tsx [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+    ],
+  },
+  {
+    type: Section.BasicContent as Section.BasicContent,
+    sections: [
+      {
+        title: "Who is it for?",
+        details: [
+          "Projects who want access to funds without selling their native tokens.",
+          "Treasuries who want to borrow stablecoins against a project token without the immediate regulatory implications of conducting a sale.",
```
Comment (Tommy1231232) [inline pages/products/range-tokens.tsx:25]: Per designs, lets have "regulatory" be on the second line

File: pages/products/lsp.tsx [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+          "Engineers that want to create Covered Call Options whereby the contract pays out if a settlement price is above or below a defined strike price.",
+          "Projects interested in creating structured products enabling them to borrow stablecoins against their project’s native token while offering unique risk profiles to investors.",
+        ],
+      },
+      {
+        title: "How is it better?",
+        details: [
+          "Compared to EMP contracts on UMA, the LSP contracts are non-liquidatable and do not require running bots.",
+          "The dual token design means that sponsor positions are transferable.",
+          "Accommodates various financial payout types and products with minimal changes",
```
Comment (Tommy1231232) [inline pages/products/lsp.tsx:35]: Lets put a period at the end of this

**Review by Tommy1231232 — CHANGES_REQUESTED** [review]

**Review by Tommy1231232 — APPROVED** [review]

**Pull request merged**